src/App.js

```diff
@@ -3,6 +3,7 @@ import React from "react";
 import Home from "./pages/Home/Home";
 import About from "./pages/About/About";
 import Skills from "./pages/SKills/Skills";
+import ProjectsPage from "./pages/Projects/ProjectsPage";
```

Comment:
are you planning to name everything with the "Page" suffix? if so it's fine that you name this ProjectsPage, otherwise i would name it just Projects for consistency's sake